### DISCUSSION TITLE
Issue with QInput of type date

### DISCUSSION THREAD

**Discussion by jonmor51** [quasarframework/quasar#12980] [General - Components / Directives / etc]
I am using a QInput of type 'date', thus:
`<q-input
      ref="signedUp"
      type="date"
      v-model="customer.signedUp"
      label="Signed Up" stack-label>
    </q-input>`
The v-model, customer.signedUp, is a Javascript date, and it is not displaying correctly, and I'm seeing errors like this in the browser console: `The specified value "2022-03-22T00:00:00Z" does not conform to the required format, "yyyy-MM-dd".`. I can select a date OK, but I just can't initialize the date picker OK. So how do I use this with a Javascript date? Do I have to use a computed property with set and get for the model, or is there some way I can format the date for use? I see no mention of this in the docs.

**Comment by mJeromeDiaz** [comment]
Hi @jonmor51, you can easily transform your date with quasar date Utils https://quasar.dev/quasar-utils/date-utils

**Reply by jonmor51** [reply]
Thanks. I know I can do this. I am actually using dayjs in this application. My point was whether it was possible somehow to us a native Javascript date with this without doing the necessary formatting - given that it is a component for handling and displaying dates, I assumed I could and I had just missed some prop or other which would do this. As this is apparently not the case, I have switched to using a computed property with a get and set, the get applying the necessary formatting.
